ui/page/wallet_page.go

```diff
@@ -568,8 +603,11 @@ func (pg *WalletPage) layoutWatchOnlyWallets(gtx layout.Context) D {
 					)
 				}),
 				layout.Rigid(func(gtx C) D {
+					pg.listLock.Lock()
+					listItems := pg.listItems
+					pg.listLock.Unlock()
```

Comment:
Use the listItems declared earlier https://github.com/planetdecred/godcr/blob/09cabbcfc0037772251c4648f00dfcf412d6dd81/ui/page/wallet_page.go#L566